nix/solidity-package.sh

```diff
@@ -1,6 +1,7 @@
 source $stdenv/setup
 unpackPhase
 
+IFS=" " read -r -a opts <<<"$solcFlags"
```

Comment:
Who defines `solcFlags`? Does it make sense to convert it to an array? We could simply pass it to `solc` as a string.

Reply:
It would be set as an implicit argument to `solidityPackage`. Maybe add `solcFlags ? ""` to make it clear that it could be set.

It could definitely be just a regular string.